src/reference/antora/modules/ROOT/pages/zeromq.adoc

```diff
@@ -119,6 +119,12 @@ If the `receiveRaw` option is set to `true`, a `ZMsg`, consumed from the socket,
 Otherwise, an `InboundMessageMapper` is used to convert the consumed data into a `Message`.
 If the received `ZMsg` is multi-frame, the first frame is treated as the `ZeroMqHeaders.TOPIC` header this ZeroMQ message was published to.
 
+If the `unwrapTopic` option is set to `false`,
+the incoming message is considered to consist of two frames: the topic and the ZeroMQ message.
+Otherwise, by default, the `ZMsg` is considered to consist of three frames:
+the first one containing the topic, the last frame containing the message,
+with an empty frame in the middle.
```

Comment:
We try to follow rule `One sentence per line`: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line